test/services/return-requirements/fetch-licence-purposes.service.test.js

```diff
@@ -14,9 +14,9 @@ const LicenceVersionPurposeHelper = require('../../support/helpers/licence-versi
 const PurposeHelper = require('../../support/helpers/purpose.helper.js')
 
 // Thing under test
-const FetchPurposesService = require('../../../app/services/return-requirements/fetch-purposes.service.js')
+const FetchPurposesService = require('../../../app/services/return-requirements/fetch-licence-purposes.service.js')
```

Comment:
In VSCode, highlight the word, hit `fn+F2` then type `FetchLicencePurposes` and watch the magic happen!

```suggestion
const FetchLicencePurposesService = require('../../../app/services/return-requirements/fetch-licence-purposes.service.js')
```